lib/app/configuration/defaultHooks.js

```diff
@@ -7,14 +7,6 @@
 module.exports = [
   'moduleloader',
   'logger',
-  'request',
   'orm',
-  'views',
-  'blueprints',
-  'responses',
-  'controllers',
-  'policies',
-  'services',
-  'userconfig',
-  'http',
+  'userconfig'
```

Comment:
I tried to – if I understand correctly, this is needed to pass postgres config via Sails to Waterline, so without it, Waterline breaks. Is my understanding incorrect?
Thanks